packages/dd-trace/src/log.js

```diff
@@ -1,27 +1,22 @@
 'use strict'
 
 const _default = {
-  debug: message => console.log(message), /* eslint-disable-line no-console */
-  error: err => console.error(err) /* eslint-disable-line no-console */
+  debug: (...args) => console.log(...args), /* eslint-disable-line no-console */
+  info: (...args) => console.error(...args), /* eslint-disable-line no-console */
+  warn: (...args) => console.error(...args), /* eslint-disable-line no-console */
+  error: (...args) => console.error(...args) /* eslint-disable-line no-console */
 }
 
 // based on: https://github.com/trentm/node-bunyan#levels
 const _logLevels = {
   'debug': 20,
+  'info': 30,
```

Comment:
This was not in the tracer logging RFC. The idea of omitting it is that it was assumed that the default logging level would be `info` and for most cases we didn't want to log in that case. This was mostly for other tracers but would apply to JS as well if we were to enable logging by default, which this PR is the start of.

@brettlangdon Should this level be added to the RFC or removed from the tracer instead?